druid-shell/src/keyboard.rs

```diff
@@ -419,19 +485,21 @@ impl From<i32> for KeyCode {
 
         match src {
             VK_LSHIFT => KeyCode::LeftShift,
+            VK_SHIFT => KeyCode::LeftShift,
```

Comment:
I'd be happy to merge all of these changes immediately; it might be worth opening a new PR for this?